### DISCUSSION TITLE
Debug on MacOS

### DISCUSSION THREAD

**Discussion by zichunxx** [google-deepmind/mujoco#2054] [Asking for Help]
### Intro

Hi!

I'm new to MacOS. 

### My setup

Mujoco: 3.2.2
MacOS: 14.6

### My question

I found that running the Python script containing `launch_passive` requires `mjpython` on MacOS. 

In this situation, how can I configure the `launch.json` file in VSCode to debug the Python script? 

Thanks for your help!

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by zichunxx** [comment]
Sorry for the careless. 

Just configure `"python": Path/to/mjpython`.

**Reply by sadelello** [reply]
I am new to MacOS as well, I pip installed Mujoco, and everything works fine, I can call Mujoco functions no problem. But when I try to use the passive viewer with mjpython, I get a "zsh: command not found: mjpython" error. Any experience with this or any idea of how to fix?
